hello-world-apk-overrides/readme.md

```diff
@@ -0,0 +1,38 @@
+# Purpose
+
+This project is an identical clone of `hello-world-apk` module and designed to show you how to override the default values
```

Comment:
I am thinking that we could simply apply the override modifications to the hello-world-apk project itself, which was intended to showcase this kind of configuration, instead of adding a second APK project.